src/Framework/Framework/Binding/Expressions/ValueBindingExpression.cs

```diff
@@ -95,7 +101,11 @@ public class OptionsAttribute : BindingCompilationOptionsAttribute
                         return new GlobalizeResourceBindingProperty();
                     }
                     return null;
-                })
+                }),
+                new Func<ValueBindingExpression, ParsedExpressionBindingProperty, ReferencedViewModelPropertiesBindingProperty>((binding, expr) =>
+                {
+                    return BindingPropertyResolvers.GetReferencedViewModelProperties(binding, expr);
+                }),
```

Comment:
I think this is not needed, it should automatically discover the `GetReferencedViewModelProperties` method